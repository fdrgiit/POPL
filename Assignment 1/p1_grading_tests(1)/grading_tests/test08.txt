
VAR VARASSIGN

> > >> << <> -+=:,02_x1111 

XYZ.1INT[101](ABCDx16.x122INT)

STRING1234...INT

PRINT.123.456INT

hello.11111111111111111111STRING

