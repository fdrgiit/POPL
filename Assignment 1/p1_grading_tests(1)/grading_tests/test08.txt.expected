2 1 VAR
2 37 VARASSIGN 
4 27 >
4 27 >
4 32 >>
4 31 <<
4 26 <>
4 14 -
4 13 +
4 18 =
4 19 :
4 20 ,
4 34 0 
4 34 2 
4 15 _
4 37 x1111 
6 35 XYZ.1INT 
6 22 [
6 34 101 
6 23 ]
6 24 (
6 37 ABCDx16 
6 33 .
6 37 x122INT 
6 25 )
8 37 STRING1234 
8 33 .
8 33 .
8 33 .
8 10 INT
10 9 PRINT
10 33 .
10 34 123 
10 33 .
10 34 456 
10 10 INT
12 37 hello 
12 33 .
12 34 11111111111111111111 
12 12 STRING
